### PR TITLE
Fix wrong DAMAGE_PITFALL and DAMAGE_DEVIL descriptions

### PR DESCRIPTION
**Pull request by OciXCrom**
DAMAGE_PITFALL description was duplicated from DAMAGE_TIMER.
DAMAGE_DEVIL is not called when getting items from the devil deal; I didn't manage to find out when it is actually used.

### CODE REVIEW

File: group__enums.html [hunk 1]
```diff
@@ -4726,9 +4726,9 @@ <h2 class="memtitle"><span class="permalink"><a href="#gab084d3a6d6ddd144ea6df4a
 </td></tr>
 <tr><td class="fieldname"><a id="ggab084d3a6d6ddd144ea6df4af965f93fca1fd9f2d48998b0eb519f4aef9e26784e"></a>DAMAGE_POOP&#160;</td><td class="fielddoc"><p>damage from red poop </p>
 </td></tr>
-<tr><td class="fieldname"><a id="ggab084d3a6d6ddd144ea6df4af965f93fca62cf902c707d3c6e4c4d128610348845"></a>DAMAGE_DEVIL&#160;</td><td class="fielddoc"><p>damage comes from devil room deal </p>
+<tr><td class="fieldname"><a id="ggab084d3a6d6ddd144ea6df4af965f93fca62cf902c707d3c6e4c4d128610348845"></a>DAMAGE_DEVIL&#160;</td><td class="fielddoc"><p> </p>
 </td></tr>
-<tr><td class="fieldname"><a id="ggab084d3a6d6ddd144ea6df4af965f93fca02369e68e3333e33c832c31096a4f8d2"></a>DAMAGE_ISSAC_HEART&#160;</td><td class="fielddoc"><p>Indicates the damage has been redirected from <a class="el" href="namespace_isaac.html">Isaac</a>'s Heart familiar </p>
+<tr><td class="fieldname"><a id="ggab084d3a6d6ddd144ea6df4af965f93fca02369e68e3333e33c832c31096a4f8d2"></a>DAMAGE_ISSAC_HEART&#160;</td><td class="fielddoc"><p>indicates the damage has been redirected from <a class="el" href="namespace_isaac.html">Isaac</a>'s Heart familiar </p>
```
Comment (wofsauge) [inline group__enums.html:4731]: Please revert this change. All lines need to start with a capital letter

File: group__enums.html [hunk 2]
```diff
@@ -4726,9 +4726,9 @@ <h2 class="memtitle"><span class="permalink"><a href="#gab084d3a6d6ddd144ea6df4a
 </td></tr>
 <tr><td class="fieldname"><a id="ggab084d3a6d6ddd144ea6df4af965f93fca1fd9f2d48998b0eb519f4aef9e26784e"></a>DAMAGE_POOP&#160;</td><td class="fielddoc"><p>damage from red poop </p>
 </td></tr>
-<tr><td class="fieldname"><a id="ggab084d3a6d6ddd144ea6df4af965f93fca62cf902c707d3c6e4c4d128610348845"></a>DAMAGE_DEVIL&#160;</td><td class="fielddoc"><p>damage comes from devil room deal </p>
+<tr><td class="fieldname"><a id="ggab084d3a6d6ddd144ea6df4af965f93fca62cf902c707d3c6e4c4d128610348845"></a>DAMAGE_DEVIL&#160;</td><td class="fielddoc"><p> </p>
```
Comment (wofsauge) [inline group__enums.html:4729]: why the removal?

**Review by wofsauge — CHANGES_REQUESTED** [review]

File: group__enums.html (hunk 1 above)
Comment (OciXCrom) [reply group__enums.html:4731]: But this is the only one that doesn't in the entire list. I did it for consistency. Do I make all others with a capital as well?

File: group__enums.html (hunk 2 above)
Comment (OciXCrom) [reply group__enums.html:4729]: Like I mentioned in the PR itself, this is not true. I tested it via debugging and didn't manage to trigger the DAMAGE_DEVIL flag when taking items from the devil deal. If you have more information on this, please share.

File: group__enums.html (hunk 1 above)
Comment (wofsauge) [reply group__enums.html:4731]: Yes please :)

File: group__enums.html (hunk 2 above)
Comment (wofsauge) [reply group__enums.html:4729]: Ok. When im back from vacation I can take a look at it. So maybe next week.

Comment (OciXCrom) [reply group__enums.html:4729]: Sounds good.

File: group__enums.html (hunk 1 above)
Comment (OciXCrom) [reply group__enums.html]: Done.

**Review by wofsauge — APPROVED** [review]
